lms/static/js/student_account/components/StudentAccountDeletion.jsx

```diff
@@ -73,7 +73,7 @@ export class StudentAccountDeletion extends React.Component {
     );
 
     const acctDeletionWarningText = StringUtils.interpolate(
-      gettext('{strongStart}Warning: Account deletion is permanent.{strongEnd} Please read the above carefully before proceeding. This is an irreversible action, and {strongStart}you will no longer be able to use the same email on edX.{strongEnd}'),
+      gettext('{strongStart}Warning: Account deletion is permanent.{strongEnd} Please read the above carefully before proceeding. This is an irreversible action, and {strongStart}you will no longer be able to use the same email on Lagunita.{strongEnd}'),
```

Comment:
Hard-coded, huh? ;)

What about generalizing it to remove _any_ specific platform name?

"you will no longer be able to use the same email _on_ _this_ _site_."

Reply:
@caesar2164 or even "the same email _here_."